web/packages/build/package.json

```diff
@@ -10,28 +10,28 @@
     "directory": "packages/build"
   },
   "dependencies": {
-    "@babel/core": "^7.26.10",
-    "@babel/preset-env": "^7.26.9",
-    "@babel/preset-react": "^7.26.3",
-    "@babel/preset-typescript": "^7.27.0",
-    "@eslint/js": "^9.23.0",
-    "@swc/core": "^1.11.15",
-    "@swc/plugin-styled-components": "^7.1.0",
+    "@babel/core": "^7.27.1",
+    "@babel/preset-env": "^7.27.1",
+    "@babel/preset-react": "^7.27.1",
+    "@babel/preset-typescript": "^7.27.1",
+    "@eslint/js": "^9.26.0",
+    "@swc/core": "^1.11.24",
```

Comment:
```
   The following dependencies have incompatible licenses:
  pnpm-lock.yaml » @swc/core@1.11.24 – License: Apache-2.0 AND LicenseRef-scancode-unknown-license-reference
  pnpm-lock.yaml » cookie-signature@1.2.2 – License: LicenseRef-scancode-unknown-license-reference AND MIT
```

idk what's up with that, both deps appear to have valid licenses in their package.json.

* https://www.npmjs.com/package/@swc/core/v/1.11.24?activeTab=code
* https://www.npmjs.com/package/cookie-signature/v/1.2.2?activeTab=code

Reply:
Added exemptions for them.